From 27be8064df8c2d95ccf7ebb5d084e9272fdb0736 Mon Sep 17 00:00:00 2001
From: Haibo Chen <haibo.chen@nxp.com>
Date: Fri, 29 Apr 2016 10:24:58 +0800
Subject: [PATCH 1035/5242] MLK-12734 ARM: dts: add TSC support for
 imx6ull-ddr3-arm2 board.

commit  c985560a1380f31358f4cc20b27ff2b5d8acef4e from
https://source.codeaurora.org/external/imx/linux-imx.git

Add touch support for the LCD screen on imx6ull-ddr3-arm2 board.
Due to the touch pin conflict with usbotg1 and I2C1 bus, so this
patch disable I2C1 bus and usbotg1.

Signed-off-by: Haibo Chen <haibo.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../arm/boot/dts/imx6ull-14x14-ddr3-arm2-lcdif.dts |   22 ++++++++++++++++++++
 1 file changed, 22 insertions(+)

diff --git a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-lcdif.dts b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-lcdif.dts
index 40e86a4..1f12d12 100644
--- a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-lcdif.dts
+++ b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-lcdif.dts
@@ -27,3 +27,25 @@
 &pwm1 {
 	status = "okay";
 };
+
+&i2c1 {
+	status = "disabled";
+};
+
+&reg_usb_otg1_vbus {
+	pinctrl-0 = < >;
+	gpio = < >;
+};
+
+&usbotg1 {
+	status = "disabled";
+};
+
+&tsc {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_tsc>;
+	status = "okay";
+	xnur-gpio = <&gpio1 3 0>;
+	measure_delay_time = <0xfff>;
+	pre_charge_time = <0xffff>;
+};
-- 
1.7.9.5

